### PR TITLE
SW-1819 Remove some console errors from web-app

### PR DESCRIPTION
**Pull request by karthikbtf**
- this could be a never-ending effort, going to do this in smaller chunks

### CODE REVIEW

File: src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx [hunk 1]
```diff
@@ -373,7 +373,7 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
         scrolled={true}
       >
         <Grid container item xs={12} spacing={2} textAlign='left'>
-          <Grid xs={12} padding={theme.spacing(1, 3, 1, 5)}>
+          <Grid item xs={12} padding={theme.spacing(1, 3, 1, 5)}>
```
Comment (constanzauanini) [inline src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx:376]: Adding this "item" is making all rows to be at the same level of the grey box of observations. We should add more padding to this rows

Comment (karthikbtf) [reply src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx:376]: Yes good point, I will revisit this.

**Comment by karthikbtf** [comment]
Will revisit this later.

File: src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx:376]: cc @joshbuchea

**Pull request closed without merging**